index.html

```diff
@@ -34,6 +34,144 @@ <h2 class="connect">LET'S CONNECT</h2>
+                    <span class="fiftin">2015</span>
+                </div>
+                <article class="art">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </article>
+                <ul class="frontend">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="seeproject" type="button">See Project</button>
+            </div>
+        </div>
     </section>
+    <section class="about">
+        <div class="aboutmyself">
+            <h2 class="heading">About<br> Myself</h2>
+            <article class="main">
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </article>
+            <h2 class="connect">LET'S CONNECT</h2>
+            <div>
+                <ul class="social">
+                    <li><a href="#"><img src="icons/Vector.png" alt="twitter_icon"></a></li>
+                    <li><a href="#"><img src="icons/Linkedin icon.png" alt="Linkedin_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (1).png" alt="Medium_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (2).png" alt="Github_icon"></a></li>
+                    <li><a href="#"><img src="icons/Vector (3).png" alt="Chill_icon"></a></li>
+                </ul>
+            </div> 
+            <button type="button" class="seeproject">Get my resume</button>   
+        </div>
+        <ul id="about">
+            <li class="languages">
+                <ul class="skillss">
+                    <li><h4>Languages</h4></li>
+                    <li><img src="icons/un.png" alt="un.png"></li>
+                </ul>
+            </li>
+            <li class="prlanguages">
+                <ul>
+                    <li><img src="icons/Ellipse 1.png" alt="Ellipse">Javascript</li>
+                    <li><img src="icons/Ellipse 1 (1).png" alt="Ellipse">HTML</li>
+                    <li><img src="icons/Ellipse 1 (2).png" alt="Ellipse">CSS</li>
+                </ul>
+            </li>
+            <li class="languages">
+                <ul class="skillss">
+                    <li><h4>Frameworks</h4></li>
+                    <li><img src="icons/u.png" alt="un.png"></li>
+                </ul>
+            </li>
+            <li><img class="separa" src="icons/Separator Bottom.png" alt="Separator"></li>
+            <li class="languages">
+                <ul class="skillss">
+                    <li><h4>Skills</h4></li>
+                    <li><img src="icons/u.png" alt="un.png"></li>
+                </ul>
+            </li>
+            <li><img class="separa" src="icons/Separator Bottom.png" alt="Separator"></li>
```

Comment:
- [ ] In the process of nesting these items, I observed that your UI is a bit messed up, kindly re-adjust it. ℹ️ 

> |     | Your Design before review                                 | Your Design after review                                    |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     | <img width="273" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235214526-b7b5e611-4a36-4a47-b35d-32fe76738288.png"> | <img width="280" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235214123-dd070a5e-302e-41fb-b3d0-0561aa7c67e3.png"> |